ttnctl/cmd/devices_set.go

```diff
@@ -204,5 +213,6 @@ func init() {
 	devicesSetCmd.Flags().Int32("altitude", 0, "Set altitude")
 
 	devicesSetCmd.Flags().String("description", "", "Set Description")
-	devicesSetCmd.Flags().StringArray("custom-attr", []string{}, "Set a custom attribute")
+	devicesSetCmd.Flags().StringArray("attr-add", []string{}, "Add an additional attribute (ex:--attr-add <name>:<value>)")
```

Comment:
If we simply call them attributes it's confusing
I mean this attributes are like the far west. I can of course be something pretty logical like *perOfBattery* but I can be *doesTheSunTurnAroundTheEarth=cookie*.
There is no control over it and people can put whatever stuff they want and it won't always be a *device attribute*. Maybe use this data to have a teaParty on the sun, why not.
I'll rename it to *key* or *meta*. 

P.S: It just occurred to me that *attribute* might not be the name we want to give to this.